android/modules/filesystem/src/java/ti/modules/titanium/filesystem/FilesystemModule.java

```diff
@@ -81,6 +81,10 @@ public boolean isExternalStoragePresent()
 	@Kroll.method
 	public FileProxy getFile(KrollInvocation invocation, Object[] parts)
 	{
+		//If directory doesn't exist, return
+		if (parts[0] == null) {
```

Comment:
What if any of the parts are null? Does it cause an NPE or end up with a nonsense "null" string as a path segment?